### PR TITLE
Ipay 283 create tabs component for cp

### PR DESCRIPTION
**Pull request by kushhsuk**
Run `yarn` then `yarn storybook` to test Tabs component

### CODE REVIEW

**Review by CodyMoore240 — DISMISSED** [review]

**Comment by VagishVela** [comment]
@Kush-Kurense Is this still a draft?

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  
+}: TabsProps) => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Box sx={{ width: '100%' }}>
```
Comment (CodyMoore240) [inline src/components/Tabs/Tabs.tsx:52]: Will this allow me to pass in new props and use the component how I would like to use it or will it always have the 3 tabs?

Comment (VagishVela) [reply src/components/Tabs/Tabs.tsx:52]: @Kush-Kurense I think what Cody was trying to say was can we have it so we can pass in the three tab names as a prop? Let me know if you want help doing this. Happy to help. 😄

**Comment by VagishVela** [comment]
@kushhsuk Can we close this?
@RobinDaugherty @kushhsuk should we archive this project?

**Comment by VagishVela** [comment]
Closing as we don't use this

**Pull request closed without merging**